client/store/auth.js

```diff
@@ -49,6 +51,15 @@ export const logout = () => {
   };
 };
 
+export const redirect = () => {
+  history.push(`/singleItem/1`)
+  console.log("in the redirect function");
+  console.log("the state",state)
```

Comment:
what's happening here?

Reply:
my attempt to get the search button working ... it's not fully working and i need to keep working on it